### DISCUSSION TITLE
Cluster based analysis

### DISCUSSION THREAD

**Discussion by dpoursanidis** [rifqiharrys/sdb_gui#2] [Ideas]
What about including a cluster-based regression sensu https://maloof.princeton.edu/ ?

**Comment by rifqiharrys** [comment]
Hi,
Which method are you referring to? I'm using methods that are available in [Scikit Learn](https://scikit-learn.org) module. And to my understanding, the suitable method of deriving bathymetric depth is using regression method rather than clustering method.

**Reply by dpoursanidis** [reply]
For sure regression but prior that, an unsupervised classification of the given image, then per class regression and in the end a weighted ensemble of the final sdb per class, based on the distance of the unsupervised classfication. By that you can eliminate the influence of the dark bottoms against the brights.

**Reply by rifqiharrys** [reply]
No, predicting bathymetric values is clearly a regression case, not classification nor clustering.
